diagnostic.md

```diff
@@ -54,7 +68,20 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+```CSS
+  .big {
+  } elements that belong to the _class_ `big`
+
+  #contentPane{
+   } element matching _ID_ `contentPane`
 
+  #important #red {
+  } elements belonging to both the `important` and `red` _classes_
+
+  div p {
```

Comment:
the space between selectors act as a descendent relationship. But this is a parent/child relationship; big difference. So the correct syntax is `div > p`.